error: implicit instantiation of undefined template 'std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >'
--- clustalW/general/VectorOutOfRange.h.orig	2010-11-17 04:12:30.000000000 -0600
+++ clustalW/general/VectorOutOfRange.h	2018-10-23 02:30:15.000000000 -0500
@@ -5,6 +5,7 @@
  */
 #include <stdexcept>
 #include <exception>
+#include <string>
 namespace clustalw
 {
 
